index.bs

```diff
@@ -170,19 +182,68 @@ When an image item is of type <dfn value export for="AV1 Image Item Type">av01</
+
+If an [=AV1 Multi-Layered Image Item=] is not associated with an 'lsel' property, the 'ispe' property shall describe the dimension of the output image of the operating point whose index is 0. Renderers are free to render either: only the output image of the [=Operating Point=] matching best its decoding capabilities, or to render all output images of all the [=Operating Points=] processed, resulting in a form of progressive decoding.
+
+NOTE: The dimensions of the final or intermediate output images might not match the ones given in the 'ispe' property. Renderers are expected to scale the output image to match the 'ispe' property.
+
+NOTE: When such a progressive decoding is not desired (e.g. for multi-view images) or when a controlled rendering of the different output images is desired (e.g. grid of multi-layered images), multiple [=AV1 Image Items=] sharing the same [=AV1 Image Item Data=] can be created and associated with different 'lsel' properties, each with a different value of 'layer_id'. In the case of an image grid composed of 4 layered images, each made of 2 operating points, controlled rendering could be achieved by creating a set of items as follows: 4 items each corresponding to the low quality/resolution image of the first operating point of the 4 multi-layered images, i.e. with 'layer_id' set to 0; 4 items corresponding the second operating point, i.e. with 'layer_id' set to 1; a grid item composing the first set of 4 images; another grid item composing the second set of 4 images.
+
+<h5 id ="layered-image-selector-property">Layered Image Indexing Property</h5>
+
+<h6 id ="layered-image-selector-property-definition" class="no-toc">Definition</h6>
+
+<pre class="def">
+  Box Type:       <dfn value export for="AV1LayeredImageIndexingProperty">a1lx</dfn>
+  Property type:  Descriptive item property
+  Container:      <code>ItemPropertyContainerBox</code>
+  Mandatory:      No
+  Quantity:       One
+</pre>
+
+<h6 id ="layered-image-selector-property-description" class="no-toc">Description</h6>
+
+The <dfn>AV1LayeredImageIndexingProperty</dfn> provides the size in bytes of each layer in the image data. It enables determining the byte ranges required to process an [=Operating Point=].
+
+The [=AV1LayeredImageIndexingProperty=] property may be associated to an [=AV1 Multi-Layered Image Item=]. It shall not be marked as essential. It shall not be associated with [=AV1 Image Items=] that are not using multiple layers.
```

Comment:
Do we want to make the property more than just descriptive? We could make the presence of it indicate that the encoder thinks progressive refinement should be done if applicable.

Something like this:
`[lsel,a1lx]`: Decoder should decode the operating point/layer specified by `lsel` and do progressive refinement (if applicable) if that has not yet been downloaded
`[a1lx]`: Same as `[lsel,a1lx]` with an implicit `lsel` = 0
`[lsel]`: Decoder should decode the operating point/layer specified by `lsel`. Progressive refinement should **not** be done. (The other layers may not be the same image.)
`[]`: Same as `[lsel]` with an implicit `lsel` = 0

Reply:
Conclusion of the meeting was that this would probably just complicate things.